notebooks/MAST/K2/K2_FFI/beginner_how_to_use_ffi.ipynb

```diff
@@ -0,0 +1,221 @@
+    "\n",
+    "This tutorial will refer to a couple K2-related terms that we define here.\n",
+    "* Campaign = During the K2 mission, the Kepler telescope observed the sky in a given pointing along the ecliptic plane for approximately 80 days at a time.  Each of these regions is referred to as a \"Campaign\", starting with Campaign 0 and ending with Campaign 19. There was also a special \"Engineering\" Campaign before Campaign 0 that lasted ~10 days.\n",
+    "* HDU = Header Data Unit.  A FITS file is made up of HDUs that contain data and metadata relating to the file. The first HDU is called the primary HDU, and anything that follows is considered an \"extension\", e.g., \"the first FITS extension\", \"the second FITS extension\", etc.\n",
+    "* BJD = Barycentric Julian Date, the Julian Date that has been corrected for differences in the Earth's position with respect to the Solar System center of mass.\n",
+    "* BKJD =  Barycentric Kepler Julian Date, the timestamp measured in BJD, but offset by 2454833.0.  I.e., BKJD = BJD - 2454833.0\n",
+    "* WCS = World Coordinate System, A FITS convention used to store coordinate information inside FITS headers.  For K2 full frame images, it is used to provide the translation needed to go from pixel coorindates to celestial coordinates in right ascension and declination."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Obtaining The Full Frame Image\n",
+    "\n",
+    "We will read the calibrated full frame image from Campaign 1 using the MAST URL location.  So that we can get started with understanding the file contents without reviewing how to automatically search for and retrieve K2 files, we won't show how to search and retrieve K2 FFIs in this tutorial."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# For the purposes of this tutorial, we just know the MAST URL location of the file we want to examine.\n",
+    "fits_file = \"https://archive.stsci.edu/missions/k2/ffi/ktwo2015092174954-c04_ffi-cal.fits\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Understanding The FFI FITS File Structure\n",
+    "\n",
+    "K2 FFI FITS files contain a primary HDU with metadata stored in the header.  There are then 21 extensions that contain more metadata in the header and stores the full frame image for that module+channel.  Note that all 21 modules are included in the FFI FITS file, even if some of them do not have image data.  Let's examine the structure of the FITS file using the astropy.fits `info` function, which shows the FITS file format in more detail.\n"
```

Comment:
There appear to be 84 more extensions?

Reply:
Correct, edited to explain there are 84 extensions (21 modules times 4 channels per module).